kwc-auth.html

```diff
@@ -26,306 +26,424 @@
 <dom-module id="kwc-auth">
     <template>
         <style>
+            .form-error {
+                text-align: left;
+                color: #ED5F5F;
+                font-weight: bold;
+                margin: auto 2px 5px 2px;
+            }            
             :host {
-                --paper-spinner-color: #ff842a;
+                font-family: var(--font-body);
             }
-            #pager {
-                background-color: #fff;
+            .bold {
+                font-weight: bold;
             }
-            #pager > * {
-                margin: auto 50px;
+            input[type=submit]:hover {
+                background-color: #fd621e;
+            }
+            input[type=submit]:disabled,
+            input[type=submit]:disabled:hover {
+                background-color: #ccc;
+                cursor: default;
+            }
+            .fields input:focus {
+                border-color: #1093F5;
             }
             h2 {
+                color: #FF6900;
+                font-size: 19px;
+                text-align: center;
+            }
+            #pager .fields span{
+                font-size: 14px;
+                color: #9FA4A8;
+                font-weight: bold;
+                letter-spacing: 0.25px;
```

Comment:
pixel definitions under 1px can be interpreted differently on different browsers. Better to use em, that way if font-size changes letter-spacing will change with it. This would be 0.17em